packages/react-stylesheets/src/StylesheetProvider.tsx

```diff
@@ -17,6 +17,7 @@ export const StylesheetProvider = (props: React.PropsWithChildren<StylesheetProv
     }),
     // Only recompute the context to pass down if the parent passes a new one. Props should not
     // be mutating dynamically for a provider, or may be doing so accidentally. Avoid recomputations.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I disabled a rule there, however I don't that `useMemo()` does the right job there